app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/MessageComposerInnerState.kt

```diff
@@ -2,17 +2,15 @@ package com.wire.android.ui.home.messagecomposer
 
 import android.content.Context
 import android.net.Uri
-import androidx.compose.runtime.Composable
-import androidx.compose.runtime.getValue
-import androidx.compose.runtime.mutableStateOf
-import androidx.compose.runtime.remember
-import androidx.compose.runtime.setValue
+import androidx.compose.runtime.*
```

Comment:
I think this one has to do with the `WildcardImport` issue on `Check Codestyle` job